components/malloc_size_of/Cargo.toml

```diff
@@ -32,6 +32,7 @@ hashglobe = { path = "../hashglobe" }
 hyper = { version = "0.12", optional = true }
 hyper_serde = { version = "0.9", optional = true }
 keyboard-types = {version = "0.4.3", optional = true}
+parking_lot = "0.6"
```

Comment:
Could this be 0.8 ? parking_lot is already duped in servo and the 0.8 version is present already (0.6 isn't though so I think your lockfile changes are invalid)

Reply:
Ok